e2e/deploy/vault/vault.yaml

```diff
@@ -168,7 +168,7 @@ spec:
             MYSQL_ROOT_PASSWORD: s3cr3t
             MYSQL_PASSWORD: 3xtr3ms3cr3t
 
-  vaultEnvsConfig:
+  secretInitsConfig:
```

Comment:
```suggestion
  secretInitConfig:
```

Reply:
We changed this in the operator:
https://github.com/bank-vaults/vault-operator/blob/c50c5734f57dc8ba26add45e2babde0479826da4/pkg/apis/vault/v1alpha1/vault_types.go#L257-L259